src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+@import 'variables';
+
+.stars {
+  display: flex;
+  list-style: none;
+
+  &__star {
+    width: $star-size;
+    height: $star-size;
+    margin-top: 16px;
+    margin-right: 4px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url(../images/star.svg);
+  }
+
+  @for $i from 1 through 5 {
+    &__star--#{$i} {
+      @if $i <= 4 {
+        background-image: url(../images/star-active.svg);
+      }
```

Comment:
What if some of your cards should have 3 or 5 active stars? You should set active stars depending on the modifier to the `stars` block so you can change it dynamically